habitat-lab/habitat/datasets/rearrange/run_episode_generator.py

```diff
@@ -289,6 +300,8 @@ def get_config_defaults() -> "DictConfig":
         cfg=cfg,
         debug_visualization=args.debug,
         limit_scene_set=args.limit_scene_set,
+        limit_scene=args.limit_scene,
```

Comment:
`limit_scene_set` was an old argument. It may still be useful if we want to generate scenes from a given split without caring about the actual distribution of episodes across each scene. I added and am currently using `limit_scene` to enforce equal number of episodes per scene and for easier parallelization.

Reply:
Okay, add comments explaining this.